### PR TITLE
Update Bank account api spec

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: spec/definitions/BankAccount.yaml [hunk 1]
```diff
@@ -13,17 +13,26 @@ properties:
...
+    description: Bank's Routing Number. Required if bank account is not created from Token for BBAN account number type. This field is write-only
     type: string
   accountNumber:
     description: Bank's Account Number. Required if bank account is not created from Token. This field is write-only
     type: string
   accountType:
-    description: Banks's Account type. Required if bank account is not created from Token
+    description: Banks's Account type. Required if bank account is not created from Token for BBAN account number type.
     type: string
+  accountNumberType:
```
Comment (slavcodev) [inline spec/definitions/BankAccount.yaml]: Perhaps would be better to put before `accountNumber`

File: spec/definitions/BankAccount.yaml [hunk 2]
```diff
@@ -13,17 +13,26 @@ properties:
...
+    description: Bank's Account Number type. A valid value is basic bank account number (BBAN) or international bank account number (IBAN)
```
Comment (slavcodev) [inline spec/definitions/BankAccount.yaml]: Is it read-only or writable?

Comment (slavcodev) [reply spec/definitions/BankAccount.yaml]: Any defaults if writable?

Comment (dara123) [reply spec/definitions/BankAccount.yaml]: writable

Comment (dara123) [reply spec/definitions/BankAccount.yaml]: default to BBAN

Comment (slavcodev) [reply spec/definitions/BankAccount.yaml]: please define the defaults then

**Pull request merged**